src/lib.rs

```diff
@@ -1353,17 +1356,26 @@ where
     }
 }
 
-impl<K, V> Clone for FlurryHashMap<K, V, RandomState>
+impl<K, V, S> Clone for FlurryHashMap<K, V, S>
 where
     K: Sync + Send + Clone + Hash + Eq,
     V: Sync + Send + Clone,
+    S: BuildHasher + Clone,
 {
-    fn clone(&self) -> Self {
-        let cloned_map = FlurryHashMap::with_capacity(self.count.load(Ordering::SeqCst));
+    fn clone(&self) -> FlurryHashMap<K, V, S> {
+        // TODO: This should be created with the FlurryHashMap::with_capacity_and_hasher once available
```

Comment:
We'll have that once https://github.com/jonhoo/flurry/pull/24 lands!

Reply:
Awesome, we can wait till then I think.